src/pages/clouddocs/customizing.md

```diff
@@ -0,0 +1,101 @@
+
+    ```Curl
+ curl -X GET -u "apikey":"{apikey}" \
+ --header "Accept:application/json" \
+ "{url}/v1/cartridges/cartridge_id?version=2019-09-01"
+    ```
+
+Replace `{apikey}` and `{url}` with the actual API key and URL in all sample codes above.
+
+The _/v1/catridges_ API is the recommended way for a cartridge deployment and is compatible with the legacy _/v1/deploy_ API. In many situations, the consumer has already deployed a cartridge using the _/v1/deploy_ API and the consumer can immediately update the same cartridge using the above POST and PUT operations on the _/v1/catridges_ API to initially create and to subsequently update the cartridge deployment.
+A typical _POST /v1/catridges_ operation creates and initializes a deployment for cartridge that has never been deployed to the system. For cartridges that have been previously deployed with the _/v1/deploy_ API, the _POST /v1/cartridges_ API will create, initialize, and update an existing cartridge deployment. Subsequent redeployments of a cartridge must use the _PUT /v1/cartridges_ API.
+
+## Legacy Cartridge Deployment
+
+1.  The consumer uses the {{site.data.keyword.wh-acd_short}} Configuration Editor to create a new cartridge (or modify an existing one) and customizes the contents (artifacts) of the cartridge to their domain. After that, the consumer will **Export** the cartridge in order to save a snapshot of the cartridge.
+2.  The consumer deploys the cartridge snapshot (a zip file) to  {{site.data.keyword.wh-acd_short}} using _POST /v1/deploy_ API. In the following curl example, the consumer's cartridge file is `./my_cartridges/name_of_cartridge_file.zip`, and `update=false` means do not update the resource if it already exists. Specifying the **update=true** parameter on the deploy API to update an existing cartridge.
+
+    Replace `{apikey}` and `{url}` with the actual API key and URL in the following example. A successful cartridge deploy will result in <q>201 Created</q> if it is a new resource, or <q>200 OK</q> if `update=true` was specified and the existing resource was updated.
+
+    ```Curl
+  curl -X POST -u "apikey:{apikey}" \
+  --header “Content-Type: application/octet-stream” \
+  --header "Accept: application/json" \
+  --data-binary @./my_cartridges/name_of_cartridge_file.zip \
+  "{url}/v1/deploy?update=false&version=2018-01-17"
+```
+
+> Some large cartridge deployments can exceed the request timeout thresholds defined in the DataPower gateways (usually after 2 mins). In that event, you may receive the following error response. See [Cartridge Deployment Timeout](/docs/wh-acd?topic=wh-acd-troubleshoot#troubleshoot_deploy_timeout) for additional considerations during the deployment of large cartridges.
```

Comment:
Accuracy and presentation/formatting issue? I think we're trying to convey something about an error that may surface, but not sure it applies here. Error example missing. If text is needed, it should wrap so it isn't scrolled off to the right.